esi_leap/db/sqlalchemy/api.py

```diff
@@ -116,15 +116,18 @@ def clauses(self, field):
 
 
 # Offer
-def offer_get(offer_uuid):
+def offer_get_uuid(offer_uuid):
```

Comment:
I have a vague sense that the convention would be something more like `offer_get_by_uuid`. Also, this needs to be added to `esi_leap/db/api.py`